Solutions/Marain.UserNotifications.Specs/Marain.UserNotifications.Specs.csproj

```diff
@@ -78,4 +89,7 @@
       <CodeBehindFile>%(RelativeDir)%(Filename).feature$(DefaultLanguageSourceExtension)</CodeBehindFile>
     </SpecFlowFeatureFiles>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Update="StyleCop.Analyzers" Version="1.2.0-beta.406" />
```

Comment:
Remove this. The `Endjin.RecommendedPractices` packages automatically add this reference for you.